File: Arrays/HasDuplicate/solution-with-sort.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        sorted_nums = sorted(nums)
        i = 0
        while i < len(sorted_nums) - 1:
            if sorted_nums[i] == sorted_nums[i+1]:
                return True
            i += 1
        return False

File: Arrays/HasDuplicate/solution-with-sort.rb
# @param {Integer[]} nums
# @return {Boolean}
def contains_duplicate(nums)
    nums.sort!
    i = 0
    while i < nums.size do
        if nums[i] == nums[i+1]
            return true
        end
        i += 1
    end
    false
end